packages/kotti-ui/source/kotti-field-toggle/KtFieldToggle.vue

```diff
@@ -71,8 +70,10 @@ export default defineComponent({
 				...field.inputProps,
 				forceUpdateKey: forceUpdateKey.value,
 			})),
-			onInput: (newValue: boolean | undefined) => {
-				field.setValue(newValue ?? null)
+			onInput: () => {
+				if (props.isDisabled) return
+
+				field.setValue(!field.currentValue)
```

Comment:
```
const newValue = field.currentValue !== null ? !field.currentValue : null field.setValue(newValue) ```

that's still not correct though because newValue can be undefined and that's what we transform to null

Reply:
I see your point, this is now refactored